packages/core/strapi/src/commands/actions/admin/create-user/action.ts

```diff
@@ -34,7 +34,7 @@ interface Answers {
   confirm: boolean;
 }
 
-const promptQuestions: inquirer.QuestionCollection<Answers> = [
+const promptQuestions: inquirer.DistinctQuestion<Answers> = [
```

Comment:
I don't understand the name change here. `DistinctQuestion` implies one. Is it meant to be an array of these or should it be plural?

Reply:
I think it wants to be an array, although i find it upsetting that the type passes.